test/integration/cloud-config-aws.ini.template

```diff
@@ -16,6 +16,9 @@ aws_access_key: @ACCESS_KEY
 aws_secret_key: @SECRET_KEY
 security_token: @SECURITY_TOKEN
 aws_region: @REGION
+# aws_cleanup controls whether the environment is cleaned up after tests have completed
+# This only applies to tests that have a cleanup stage
+aws_cleanup: false
```

Comment:
I discussed this with @nitzmahone and he suggested to make the default behavior to run cleanup and have the [AWS cloud plugin](https://github.com/ansible/ansible/blob/devel/test/runner/lib/cloud/aws.py) disable the cleanup. That way it's harder for users to end up with unwanted resources because they forgot to enable cleanup or didn't know they needed to.